app/node_modules/modes/node_modules/node.js

```
@@ -221,6 +208,34 @@ var Node = oop.Class({
 		}}
 	],
 
+	statics: {
+
+		/**
+		 * Creates a new node by merging the nodes present in the array. The type of the new node will be the same as the first node in the list.
+		 * All flags and properties will be ignored
+		 * @param {Array} nodes Array of consecutive nodes to merge
+		 * @return {Node}
+		 */
+		merge: function (nodes) {
+			if (!nodes || nodes.length === 0) {
+				return nodes;
```

I think we should rather throw an error here. Still the errors/exceptions management in this application is not in place, but we should give: 
- a message
- a copy of the actual input